cmd/aro/const.go

```diff
@@ -4,9 +4,10 @@ package main
 // Licensed under the Apache License 2.0.
 
 const (
-	envDatabaseName          = "DATABASE_NAME"
-	envDatabaseAccountName   = "DATABASE_ACCOUNT_NAME"
-	envKeyVaultPrefix        = "KEYVAULT_PREFIX"
-	envOpenShiftVersions     = "OPENSHIFT_VERSIONS"
-	envInstallerImageDigests = "INSTALLER_IMAGE_DIGESTS"
+	envDatabaseName                     = "DATABASE_NAME"
+	envDatabaseAccountName              = "DATABASE_ACCOUNT_NAME"
+	envKeyVaultPrefix                   = "KEYVAULT_PREFIX"
+	envOpenShiftVersions                = "OPENSHIFT_VERSIONS"
+	envInstallerImageDigests            = "INSTALLER_IMAGE_DIGESTS"
+	envPlatformWorkloadIdentityRoleSets = "PLATFORM_WORKLOAD_IDENTITY_ROLE_SETS"
```

Comment:
Since these are being exported shouldn't they all be PascalCase? https://go.dev/ref/spec#Exported_identifiers

Reply:
These consts are only used in package `main`, so I don't see a reason to export them.

Reply:
Ahh. It's package scope not file scope